### PR TITLE
<Checkbox /> - various bug fixes:

### PR DESCRIPTION
**Pull request by Faradey27**
### What changed

 - Remove active prop
 - Remove large size

### Why it changed

https://github.com/wix-private/wsr-issues/issues/67

---

- [x] Change is tested
- [x] Change is documented
- [ ] Build is green
- [ ] Change of UX is approved by [wuwa](https://github.com/wuwa) or [milkyfruit](https://github.com/milkyfruit)

### CODE REVIEW

**Comment by Faradey27** [comment]
@shlomitc should size property be removed? or extended with small?

or current way(like in pull) is fine?

File: src/Checkbox/Checkbox.js [hunk 1]
```diff
@@ -43,6 +44,15 @@ class Checkbox extends WixComponent {
     }
   };
 
+  componentDidMount() {
+    super.componentDidMount();
+    lowPriorityWarning('Checkbox prop "size" with value "large" is deprecated and will be removed in next major release, please use "medium" size instead');
```
Comment (shlomitc) [inline src/Checkbox/Checkbox.js]: Call it only if a `large` prop was passed.
Move it to the render function to cover all lifecycle hooks

File: src/utils/__test__/lowPriorityWarning.spec.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import lowPriorityWarning from './../lowPriorityWarning';
```
Comment (shlomitc) [inline src/utils/__test__/lowPriorityWarning.spec.js]: don't need this file in `__test__`, just put it next to the `lowPriorityWarning.js` file

File: src/utils/lowPriorityWarning.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
+let lowPriorityWarning = function () {};
```
Comment (shlomitc) [inline src/utils/lowPriorityWarning.js]: I would call it `deprecationLog`

Comment (Faradey27) [reply src/utils/lowPriorityWarning.js]: ok, i stole the name from React)

**Comment by Faradey27** [comment]
@shlomitc build is green again

File: src/Checkbox/Checkbox.js [hunk 4]
```diff
@@ -21,7 +22,7 @@ class Checkbox extends WixComponent {
 
   static propTypes = {
     /** used for automatic testing */
-    active: bool,
+    active: bool, // TODO remove in next major version
```
Comment (shlomitc) [inline src/Checkbox/Checkbox.js]: add a deprecation also for the `active` prop

**Comment by Faradey27** [comment]
@shlomitc green again

**Pull request merged**